classes/Refund.php

```diff
@@ -92,14 +106,17 @@ public function refundPaypalOrder()
      */
     public function getCaptureId()
     {
+        // API call here
         $response = (new PaypalOrder($this->getPaypalOrderId()))->getOrder();
 
         if (false === $response['status']) {
             return false;
         }
 
-        $purchaseUnits = current($response['purchase_units']);
-        $capture = current($purchaseUnits['payments']['captures']);
+        //@todo Quick fix before refactoring
+        $purchaseUnits = isset($response['purchase_units']) ? current($response['purchase_units']) : null;
+        //@todo Quick fix before refactoring
+        $capture = isset($purchaseUnits['payments']['captures']) ? current($purchaseUnits['payments']['captures']) : null;
         $captureId = $capture['id'];
```

Comment:
This is going to throw errors if `$capture` is null.

Reply:
Done